bootloader/common/main.c

```diff
@@ -145,6 +145,9 @@ int main(int argc, char* argv[])
         if (pyi_utils_set_environment(archive_status) == -1)
             return -1;
 
+        /* Transform parent to background process */
+        pyi_parent_to_background();
```

Comment:
the check is in the pyi_parent_tao_background. But another pull request already solved it I think! 
Thanks!